imod/mf6/gwtgwt.py

```diff
@@ -0,0 +1,89 @@
+from typing import Optional
+
+import cftime
+import numpy as np
+import xarray as xr
+
+from imod.mf6.auxiliary_variables import expand_transient_auxiliary_variables
+from imod.mf6.exchangebase import ExchangeBase
+from imod.mf6.package import Package
+from imod.typing import GridDataArray
+
+
+class GWTGWT(ExchangeBase):
+    """
+    This package is for writing an exchange file, used for splitting up a model
+    into different submodels (that can be solved in parallel). It (usually)
+    is not instantiated by users, but created by the "split" method of the
+    simulation class."""
+
+    _auxiliary_data = {"auxiliary_data": "variable"}
+    _pkg_id = "gwtgwt"
+    _template = Package._initialize_template(_pkg_id)
+
+    def __init__(
+        self,
+        transport_model_id1: str,
+        transport_model_id2: str,
+        flow_model_id1: str,
+        flow_model_id2: str,
+        cell_id1: xr.DataArray,
+        cell_id2: xr.DataArray,
+        layer: xr.DataArray,
+        cl1: xr.DataArray,
+        cl2: xr.DataArray,
+        hwva: xr.DataArray,
+        angldegx: Optional[xr.DataArray] = None,
+        cdist: Optional[xr.DataArray] = None,
+    ):
+        super().__init__(locals())
+        self.dataset["cell_id1"] = cell_id1
+        self.dataset["cell_id2"] = cell_id2
+        self.dataset["layer"] = layer
+        self.dataset["model_name_1"] = transport_model_id1
+        self.dataset["model_name_2"] = transport_model_id2
+        self.dataset["flow_model_name_1"] = flow_model_id1
+        self.dataset["flow_model_name_2"] = flow_model_id2
+        self.dataset["ihc"] = xr.DataArray(np.ones_like(cl1, dtype=int))
+        self.dataset["cl1"] = cl1
+        self.dataset["cl2"] = cl2
+        self.dataset["hwva"] = hwva
```

Comment:
You are still using the old way of initializing datasets. For example, for the ``GWFGWF`` this changed to:

```python
        dict_dataset = {
            "cell_id1": cell_id1,
            "cell_id2": cell_id2,
            "layer": layer,
            "model_name_1": model_id1,
            "model_name_2": model_id2,
            "ihc": xr.DataArray(np.ones_like(cl1, dtype=int)),
            "cl1": cl1,
            "cl2": cl2,
            "hwva": hwva,
        }
        super().__init__(dict_dataset)
```

Reply:
done